### PR TITLE
Document the docs service

### PR DESCRIPTION
**Pull request by GeorgianaElena**
Closes https://github.com/2i2c-org/pilot-hubs/issues/186

### CODE REVIEW

File: docs/configure.md [hunk 1]
```diff
@@ -191,4 +191,36 @@ For an example, see [the log-in page of the staging hub](https://staging.pilot.2
...
+
+```yaml
+  config:
+    docs_service:
+      enabled: true
+      repo: https://github.com/<static-web-files-repo-name>
+      branch: <branch>
+```
+
+**NOTE:**
```
Comment (choldgraf) [inline docs/configure.md:220]: Wanna make this a:

````
```{note}
And put the note here?
```
````

:-)

Comment (GeorgianaElena) [reply docs/configure.md:220]: Oh yeah! Thank you!I was wondering how to have a proper note :sweat_smile:

Comment (GeorgianaElena) [reply docs/configure.md]: It looks so much better now :heart_eyes:

Comment (choldgraf) [reply docs/configure.md]: go MyST markdown! 🚀

**Comment by yuvipanda** [comment]
Thanks, @GeorgianaElena!

**Pull request merged**